### ISSUE TITLE
No lr decay?

### ISSUE THREAD

**Issue by zdhNarsil** [karandwivedi42/adversarial#1]
In `main.py` I didn't see any code about learning rate decay. Is is an omission by mistake?

**Comment by karandwivedi42** [comment]
This is intended as a minimal adaptation of https://github.com/kuangliu/pytorch-cifar with adversarial training code. LR Decay and all other tricks can be added as needed.
